src/current/v23.1/load-based-splitting.md

```diff
@@ -69,6 +69,27 @@ However, by leveraging load-based splitting, the cluster can understand load on
 
 This benefit is further amplified by [load-based rebalancing]({% link {{ page.version.version }}/architecture/replication-layer.md %}#load-based-replica-rebalancing), which ensures that all nodes contain replicas with a roughly equal load. By evenly distributing load throughout your cluster, it's easier to prevent bottlenecks from arising, as well as simplifying hardware forecasting.
 
+## Monitor load-based splitting
+
+The [INFO]({% link {{ page.version.version }}/logging.md %}#info) log message
+
+~~~
+no split key found: ...
+~~~
+
+indicates that CockroachDB wants to [split the range]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#range-splits) due to load, but couldn’t find a key to split at.
```

Comment:
```suggestion
indicates that CockroachDB wants to [split the range]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#range-splits) due to load, but couldn’t find a key to split into smaller ranges.
```

Reply:
y'know I'm  hesitant to take this one, since I think it's changing the meaning slightly.  I think the key "being split at" is not the same as "the key being split".  

@kvoli can you please verify if this change is ok and/or explain if this is actually a distinction here between "to split at" vs "to split into ranges"?  my sense is there is a difference but not an expert on this

Reply:
No worries -- fyi I took that wording from our existing https://www.cockroachlabs.com/docs/stable/load-based-splitting

Reply:
The different between "to split at" and "to split into smaller ranges" is that the "to split at" implies there will be 2 ranges post split (to me at least), while "to split into smaller ranges" could imply >= 2 ranges. The log line is about splitting into 2 ranges (as with all splits), and concerns a single range we are trying to find a key to split on.

"key to split at" is more appropriate here imo, we are talking about a single split, not a series of splits into smaller ranges.

Reply:
ok thanks for clarifying this! gonna leave it then